extract/extractor_test.go

```diff
@@ -544,14 +544,14 @@ func TestExtractor(t *testing.T) {
 	`
 
 	extractor := Extractor{
-		read.ReadConfig{
-			"https://not.used.com",
-			"my-fake-space-id",
-			"my-fake-content-key",
-			"en-US",
+		ReadConfig: read.ReadConfig{
+			UsePreview:  "https://not.used.com" == "",
```

Comment:
I think this should actually be a boolean - my bad for not changing that earlier. Surprised the linter is catching that

Reply:
Lol - missed that. Yes - superconfusing